osgi-test/src/test/java/org/ehcache/osgi/SimpleOsgiTest.java

```diff
@@ -70,6 +70,7 @@ public Option[] individualModules() {
       gradleBundle("org.terracotta:statistics"),
       gradleBundle("org.ehcache:sizeof"),
       gradleBundle("org.terracotta:offheap-store"),
+      gradleBundle("org.terracotta:fast-restartable-store"),
```

Comment:
Hmmm... should the `offheap-store` manifest declare an optional dependency on `fast-restartable-store`?

Reply:
yeah that was my question when I did that... I don't know. So we have other cases of that in Echache ?

Reply:
I kinda feel like this depends on when/if we move the features FRS supports in to OSS.

Reply:
Since we don't know the answer now, what is best to do: mark is optional, and eventually later remove the optional, or mark it required from start, eventually by marking it optional later ?